packages/gensx-core/src/index.ts

```diff
@@ -1,21 +1,5 @@
-export { createContext, useContext } from "./context.js";
-export { execute, Workflow } from "./execute.js";
-export { Fragment, jsx, jsxs } from "./jsx-runtime.js";
-export type { JSX } from "./jsx-runtime.js";
-export { StreamComponent, Component } from "./component.js";
-export { array } from "./array.js";
-export type {
-  Context,
-  ComponentOpts,
-  MaybePromise,
-  Streamable,
-  ComponentProps,
-  GsxStreamComponent,
-  GsxComponent,
-  GSXToolProps,
-  GSXToolAnySchema,
-} from "./types.js";
-export type { GsxArray } from "./array.js";
-export { readConfig } from "./utils/config.js";
-export { getSelectedEnvironment } from "./utils/env-config.js";
-export { readProjectConfig } from "./utils/project-config.js";
```

Comment:
oh just realized we are still gonna need these. they're used by the storage package